meetings/2023-09-19.md

```diff
@@ -0,0 +1,140 @@
+* MdJ: What we tried to do in the [solid data modules](https://hackmd.io/@michielbdejong/HyIMjmoxn) project is to not having an idea about how an idea of storing data in a pod is good or bad, but just see what is there and what ends up in a pod. Whenever someone wants to write something in a pod, we make the data module compatible with that.
+* TBL: When we talk about how we define c2c specs there's an interesting part of Solid, when you talk about modularity. Typically the modularize around data classes: I can handle an address book, a chat etc. So in this way a chat app signs up for both long chat as 'message' data. 
+* TBL: Suppose I want to link a chat message to a task, in which case it becomes an issue. In other cases, one might take an issue and integrate it in a message... In the spec, we could add some integrations.
+* WT: My main comment is that we can abstract from that and have a general spec which lets us integrate different types of data. The app can also ask pod - do you have this kind of data?
+* TBL: ...
+* WT: One can write chat spec, someone else writes a spec for recipes. We can have general spec which defines how those can be integrated with having dedicated spec for integration between any possible domains.
+
+---
+
+* eP: Coming back a bit to the IRI template with dates, I think we should describe the requirements for that and distinguish it form other popular approaches link [ULIDs](https://github.com/ulid/spec). So what do we want to reach with it? This would also be usable more general.
+* eP: Tomorrow I also want to dmeo shortly the updated Authorization Agent, it also relates to a general way on which way the app knows where it finds data and how it get's access to that specific type of data: a common reference. I will demo how it would work for projects and tasks, and we can then see the general pattern that we would need for any kind of data.
+* TBL: If we want other things than type indexes, one thing to do is to standardize a javascript API.
+* eP: I do have a [typescript implementation](https://github.com/janeirodigital/sai-js) of [SAI](https://solid.github.io/data-interoperability-panel/specification/), which I will use in the demo. But I don't think we can rely to much on one specific programming language.
+* eP: Here's also a link to the [previous demo](https://www.youtube.com/watch?v=CvTunOWu7mM).
+* TBL: ... (something about websockets) ... That's why I'm thinking we could have abstraction in the code which helps people move from one to the other.
+
+---
+
+* TBL: I am a bit worried about things being treated the same if they are the same type. Some address books might have different needs to access. 
+* eP: I agree. But I will show on the demo tomorrow, e.g. distinguish between data from different storages. It also uses an authorization agent, which the user can trusts and easily used because of the familiarity of the UI.
```

Comment:
```suggestion
* eP: I agree. But I will show on the demo tomorrow, e.g., distinguishing between data from different storages. It also uses an authorization agent, which the user can trust and is easily used because of the familiarity of the UI.
```